src/api/searchCities.ts

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+import {
+    SearchCityTypes,
+    CIRCLE,
+    RADIUS,
+    COUNTRY_CODE,
+    LIMIT,
+    LANG
+} from '../utils/constans';
+
+import City from '../types/City/index'
+
+async function searchCities(search: string) {
+    const path = `${process.env.REACT_APP_HERE_API_BASE_URL}autocomplete`;
+    const requestConfig = {
```

Comment:
U can create default config for Axios instead of creating requestConfig every single time